CompilerSource/parsing/macros.cpp

```diff
@@ -257,7 +256,12 @@ TokenVector Macro::SubstituteAndUnroll(
           for (const Token &tok : args[ind])
             str += tok.content;
           str = QuoteString(str);
-          TokenVector vec{Token(TT_STRINGLIT, errc.snippet)};
+
+          stringified_macros.insert(str);
```

Comment:
This looks like a good place to avoid a second lookup by using the iterator returned by `insert`.

Reply:
Fixed with c1181c90e9f7734aebaad288438c0ddddf13e44e! I was not aware that `insert` returned an iterator.